### ISSUE TITLE
Don't get a message "Send to <IP>"

### ISSUE THREAD

**Issue by RaBuAc** [Snapmaker/SnapmakerJ1CuraPlugin#88]
I 'm running cura 5.6 and added the snapmaker plugin. I've selected the printer accordning to the description as a "Add a non-networked printer". 
When I slice now I do not get any choice of sending the sliced file to the snapmaker directly. 
What am I missing?
Many thanks for your help.

**Comment by firebladed** [comment]
I think i get how this works with a UDP broadcast on the subset (udp broadcast  to port 20054), but this only works on the same subnet. so could realy do with somewhere to enter a ip address for the printer directly

**Comment by bbarenz** [comment]
Agreed no option to send over network or input the IP. Searching in cura doesn't find a networked printer.

**Comment by chinloon** [comment]
similar.  unable to find a way to ad IP address for snapmaker A350 from cura
